src/TestHelper/dir.props

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@weshaggard @ericstj  what you think of naming this library? I agree with @stephentoub to have better name as TestHelper is more generic. Having CoreFX in the name can give idea it is corefx specific library.